src/components/DocumentationTopic/PrimaryContent/Mentions.vue

```diff
@@ -0,0 +1,55 @@
+<!--
+  This source file is part of the Swift.org open source project
+
+  Copyright (c) 2024 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+
+<template>
+  <section v-if="topics.length" class="mentions">
+    <LinkableHeading class="mentions" id="mentions">
```

Comment:
The linkable behavior of this doesn't seem to be working, but I think this might fix it:

```suggestion
    <LinkableHeading class="mentions" anchor="mentions">
```

Reply:
You can probably also remove the class, since it's repeating the parent class and doesn't seem to be used in any CSS.

Reply:
To test the linkable behavior, you can just click on the "Mentioned in" heading, and it should link directly to that subsection and change the URL to `#mentions`